tasks/sync-granule/schemas/config.json

```diff
@@ -2,7 +2,7 @@
   "title": "SyncGranuleConfig",
   "description": "Describes the config used by the sync-granule task",
   "type": "object",
-  "required": ["buckets", "collection", "provider", "downloadBucket"],
```

Comment:
If you want for the entire collection object to be optional, you will also need to delete the `required` field from the `collection` property, else it will complain that a required field of undefined does not exist.

Reply:
@mennovandiermen You're saying that unless the property is required it can't have required fields?

Reply:
I have had issues with this in the past, yes. JSONSchema does not validate them as required fields of an optional object, but simply as required fields, which validation rejects when the object is not provided. I don't recall exactly which schema I was working on, but here is an example of the issue: https://github.com/mozilla-services/react-jsonschema-form/issues/675 If you try running SyncGranule with this schema but no collection object provided, I think you will see the schema validation fail.

Reply:
HI guys.  I am not seeing the schema validation fail.  If we do not make that change, our SyncGranule fails because no collection is defined.  We have to make a decision ASAP so we can close the ticket.  What do you propose?